packages/strapi-plugin-i18n/admin/src/components/LocaleList/index.js

```diff
@@ -7,11 +7,25 @@ import PropTypes from 'prop-types';
 import { useLocales } from '../../hooks';
 import LocaleRow from '../LocaleRow';
 import { getTrad } from '../../utils';
+import ModalEdit from '../ModalEdit';
+import ModalDelete from '../ModalDelete';
 
-const LocaleList = ({ onAddLocale, onDeleteLocale, onEditLocale }) => {
-  const { locales, isLoading } = useLocales();
+const LocaleList = ({ canUpdateLocale, canDeleteLocale, canCreateLocale }) => {
+  const [localeToDelete, setLocaleToDelete] = useState(undefined);
```

Comment:
Why do you need to explicitly set the default value to `undefined`. Isn't that already `undefined` by default?

Reply:
Yep, you're right, it's not necessary 😊